### PR TITLE
Minor changes to installation docs

### PR DESCRIPTION
**Pull request by gherceg**
Originally added on https://github.com/dimagi/commcare-cloud/pull/5424, I didn't like mixing a pure refactor with an addition to docs. This is just the refactor of existing docs.

I found it a bit hard to follow these docs previously, and felt that adding more clearly defined sections and better organization could help so that is what is being done here.

##### ENVIRONMENTS AFFECTED

None

### CODE REVIEW

**Review by skodde — APPROVED** [review]

File: hosting_docs/source/installation/2-manual-install.rst [hunk 1]
```diff
@@ -70,112 +70,112 @@ cluster’s proxy server.
...
-      otherwise skip it.
+    If your default version is not 3.x or if the “python” command was
+    not found, make python3 your default by running the command below,
+    otherwise skip it.
 
-      ::
+    ::
 
-         $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+        $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```
Comment (millerdev) [inline hosting_docs/source/installation/2-manual-install.rst:164]: Didn't change here, but should this say
```suggestion
        $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.10
```

**Comment by sravfeyn** [comment]
👍
